### DISCUSSION TITLE
使用trt推理的时候报错：Error Code 3: API Usage Error (Parameter check failed

### DISCUSSION THREAD

**Discussion by patrickpy** [PaddlePaddle/PaddleOCR#12782] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
Ubuntu 20.04.3 LTS \n \l
TensorRT-8.4.3.1
paddlepaddle-gpu          2.4.2.post117
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
```
python tools/infer/predict_system.py --det_model_dir="output/det/ch_PP-OCR_v3_det_inference"  --rec_model_dir="output/rec_en_inference" --cls_model_dir="output/cls_inference"  --image_dir="test/test.jpg" 
```
- 完整报错/Complete Error Message：
E0823 17:46:05.654536 1869783 helper.h:114] 3: [executionContext.cpp::setOptimizationProfileInternal::781] Error Code 3: API Usage Error (Parameter check failed at: runtime/api/executionContext.cpp::setOptimizationProfileInternal::781, condition: profileIndex >= 0 && profileIndex < mEngine.getNbOptimizationProfiles()
)

**Comment by FWC13163296500** [comment]
请问该问题是否解决？
